p4-16/spec/grammar.mdk

```diff
@@ -515,7 +515,12 @@ keyElement
 
 actionList
     : /* empty */
-    | actionList actionRef ';'
+    | actionList optAnnotations actionRef ';'
+    ;
+
+actionRef
+    : prefixedNonTypeName
```

Comment:
would be nice to have the implementation use the same grammar, or at least test them in the implementation to make sure there's no subtle bug.

I think we can take this change as a bug-fix, it does not need to be a new feature in 1.3.0.

Reply:
Yep, I'll also update p4c to follow this, which seems to be the simplest way to do things (vs. having two `actionRef`-like rules, one with annotations and one without).